pkg/datapath/iptables/iptables.go

```diff
@@ -134,22 +134,32 @@ func removeCiliumRules(table, prog string) {
 
 func (c *customChain) remove() {
 	if option.Config.EnableIPv4 {
-		runProg("iptables", []string{
-			"-t", c.table,
-			"-F", c.name}, true)
+		prog := "iptables"
+		args := []string{"-t", c.table, "-F", c.name}
+		err := runProg(prog, args, true)
+		if err != nil {
+			log.WithError(err).WithField(logfields.Object, args).Warnf("Unable to flush Cilium %s chain", prog)
```

Comment:
Nit: maybe it makes sense for the sake of structured logging to introduce `logfields.IPTablesProg` and do `[..].WithField(logfields.IPTablesProg, prog).Warn("Unable to flush Cilium chain")`?

Reply:
Could be done, but there's a bunch of other usages in this file that would also need updating---this is following existing style. In this case there's only two variations of the message so it should be much better than if there was arbitrary data being stringified in the message.